module/x/gravity/keeper/attestation.go

```diff
@@ -243,7 +244,12 @@ func (k Keeper) IterateAttestations(ctx sdk.Context, reverse bool, cb func([]byt
 	} else {
 		iter = store.Iterator(prefixRange(prefix))
 	}
-	defer iter.Close()
+	defer func(iter storetypes.Iterator) {
```

Comment:
what lead to this change?

Reply:
I got a suggestion while working on this file that this error was unhandled, I can remove if you don't like